contents/handbook/growth/customer-support.md

```diff
@@ -32,46 +32,43 @@ Your objective should be to get the conversation into Zendesk ASAP, because it's
 
 ### Response Targets
 
-We have a high volume of tickets created, and some require more attention than others.  Here we have our targets for _initial_ response/acknowledgement to issues by Customer Priority and Issue [Severity](/docs/support-options#severity-levels), along with an indication of who is responsible for the initial response (CS = Customer Success, SH = designated Support Hero for the impacted team)
-
-<table>
-  <tr>
-    <th colspan="2"></th><th colspan="3"> Customer Priority</th>
-  </tr>
-  <tr>
-    <td colspan="2"></td>
-    <th>High</th>
-    <th>Medium</th>
-    <th>Low</th>
-  </tr>
-<tr>
-    <th rowspan="4">Issue Severity</th>
-    <th>Critical</th>
-    <td>1 hour (CS)</td>
-    <td>1 hour (CS)</td>
-    <td>1 day (CS)</td>
-  </tr>
-<tr>
-    <th>High</th>
-    <td>1 hour (CS)</td>
-    <td>1 day (SH)</td>
-    <td>1 day (SH)</td>
-  </tr>
-<tr>
-    <th>Medium</th>
-    <td>1 day (SH)</td>
-    <td>1 day (SH)</td>
-    <td>2 days (SH)</td>
-  </tr>
-<tr>
-    <th>Low</th>
-    <td>1 day (SH)</td>
-    <td>2 days (SH)</td>
-    <td>Automated Response (See below)</td>
-  </tr>
-</table>
-
-Note for Low Priority/Low Severity tickets we should auto-respond letting customers know that they can ask questions in Community Questions
+We have a high volume of tickets and we're a small team so we're not able to respond to all issues equally. For this reason we prioritize tickets into three categories. We set a response target for each so that we can be sure that tickets are being handled effectively. 
+
+Note that tickets are automatically prioritized in Zendesk and users are updated with information about response targets to set appropriate expectations. In all cases, tickets are routed to the appropriate team and that team is responsible for meeting the response target.
+
+The response targets listed below are our minimums, and we often respond far faster. Please note that we do not offer any level of weekend customer support.
+
+#### High priority
+**Response target: 12 hours**
+
+Tickets are considered high priority if they fulfill ANY of the following conditions:
+
+- The user is tagged as belonging to a priority customer org
+- The user is in a trial stage with the product
+- The user raises an issue through a shared Slack channel
+- The user belongs to an org which qualifies as a high-paying customer
+- The ticket is listed as critical severity
+
+This ensures that users who pay for support or which are otherwise considered a priority customer are prioritized and get the best possible support experience. Free users can raise critical impact bugs or issues to an appropriate level.
+
+#### Normal priority
+**Response target: 24 hours**
+
+Tickets are considered normal priority if they fulfill ANY of the following conditions but the user does NOT qualify as a high-paying org:
+
+- The org is a paying customer
```

Comment:
Previously this was 2-3 days. See https://github.com/PostHog/billing/pull/566/files#diff-7197565905d6831d358b3b730a09badd271aa5761dfe4e4ab330a92002706bbf

Are we sure we want this to be 24 hours? Maybe just what we had in billing was wrong, which is fine, just double checking.

Reply:
Yeah, I think the billing was wrong in this case. The handbook specified that support times fit in a rubric and varied between 1-48 hours, with us promising a 1-24 hours in 90% of cases (can see what changed on this PR). That's also what we were tracking in Zendesk and reporting against. That means our actual targets were more aggressive before this change.

The idea here is just to lessen off the targets a little bit, mainly giving us 12 hours instead of 1 hour to respond for High prio tickets and clarifying that Low priority tickets don't have assured responses. It also just simplifies things down to a clearer priority structure - no complex rubric.

Our SLA performance wasn't absolutely terrible under the old system (below) which set a 1 hour response target for some issues, so it should improve with a little more slack and we can then optimize down as we grow the team. 

<img width="649" alt="Screenshot 2024-04-25 at 09 34 40" src="https://github.com/PostHog/posthog.com/assets/84011561/b5eb7f2c-8cf7-4848-9f8c-5ed9af324ba4">

Reply:
To the second point, Marius and Paul have at least approved the proposal and I think it's Replay + Analytics who will be most affected here.

Regarding how we define paying customers, there are two criteria we use for routing.

- High paying customer is based on a forecasted MRR in excess of $1666, which is a benchmark set by CS. 
- Paying customer is defined using the `paying_customer` tag in Zendesk. I'm not 100% sure whether this means non-zero bill or a paid bill, however. @MarconLP can you confirm?

Either way, the Support team triages product analytics tickets which I think directly addresses the concern here as they'll be giving the first response in most cases.

Reply:
`paying_customer` is anyone above $0 and below $1666 per month

Reply:
I'd leave this as it is. These users can raise tickets, but we respond immediately directing them to community support and not promising a response.